src/Featurizers/SimpleRollingWindowFeaturizer.h

```diff
@@ -0,0 +1,359 @@
+// ----------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License
+// ----------------------------------------------------------------------
+#pragma once
+
+#include "../Archive.h"
+#include "../Featurizer.h"
+#include "../Traits.h"
+
+#include "Base/RollingWindowTransformerBase.h"
+#include "Calculators/MaxCalculator.h"
+#include "Calculators/MinCalculator.h"
+#include "Components/FilterDecoratorFeaturizer.h"
+#include "Components/GrainFeaturizerImpl.h"
+#include "Components/InferenceOnlyFeaturizerImpl.h"
+#include "Components/PipelineExecutionEstimatorImpl.h"
+#include "Components/WindowFeaturizerBase.h"
+
+#include <vector>
+
+namespace Microsoft {
+namespace Featurizer {
+namespace Featurizers {
+
+// Anonymous namespace for helper using statements
+namespace {
+    using GrainType = std::vector<std::string>;
+} // Anonymous namespace
+

```

Comment:
Give this namespace an actual name; if it remains anonymous, it will be recreated in every compilation unit.

Reply:
I decided to move it into the GrainFeaturizer itself as a public type.